elements/storytelling/src/markdown-it-plugin/markdown-it-decorate-improved.js

```diff
@@ -10,6 +10,8 @@ import slugify from "@sindresorhus/slugify";
  */
 export default function attributes(md) {
   md.nav = [];
+  md.customElements = ["eox-", "story-telling-"];
```

Comment:
I don't understand yet how this works. In the story and in the tests you created other custom elements (foo-bar, baz-que and quux-corge) which would theoretically not be part of the whitelist?

Maybe we can do without this whitelist here (and just require a `-` in the element tag)? If not, then maybe this whitelist config can be extracted from markdown-it-decorate-improved to be defined somewhere else.

Reply:
Another thought, wouldn't it be good to allow entirely replacing the default rendered element with any element, not just custom elements?

Example: 
```md
## Hello <!--{as="p" .custom-paragraph}--> // would replace h1 with a p tag
```

Reply:
We need to provide a custom element list or a regex pattern to identify which tag to be ignored during sanitize. Otherwise, it won't be part of HTML code. 

But you are right we can check for `-` in DOM sanitize and pass everything with `kebab-case` and gets ignored in the sanitize function.

Reply:
Done ✅